src/dataregistry/schema/schema.yaml

```diff
@@ -87,6 +87,7 @@ tables:
         type: "Boolean"
         description: "True if this keyword is active and avaliable to use"
         nullable: False
+        modifiable: True
```

Comment:
Actually we don't need this.   The `modifiable` attribute is used by `BaseTable.modify` to determine if the user can modify an attribute directly. `active` is not changed using `BaseTable.modify`, but by using the functions `disable_keyword` and `enable_keyword`.